### ISSUE TITLE
A few HTTP 500 Internal Server Error messages after enabling Activity Stream experiment via Test Pilot

### ISSUE THREAD

**Issue by pdehaan** [mozilla/embedly-proxy#70]
Found in https://embedly-proxy.stage.mozaws.net/__version__ (8682d1204c9ae5210b517d949619689589ccc268):

```json
{ "commit": "8682d1204c9ae5210b517d949619689589ccc268",
  "version": "0.5",
  "source": "https://github.com/mozilla/embedly-proxy.git" }
```

---

### Steps to reproduce:
1. Go to https://testpilot.firefox.com/experiments/activity-stream (log in w/ mozilla email).
2. Open the Browser Console (Tools > Web Developer > Browser Console).
3. Enable the Activity Stream experiment.

### Actual results:
I saw a few HTTP 500 POSTs to https://embedly-proxy.stage.mozaws.net/v2/extract

![browser_console_and_firefox_test_pilot_-_activity_stream_and_webextensions_chrome_ext_downloader_save_get_started_js_at_master_ _stuartphilp_webextensions_chrome_ext_downloader_ _github_and_software_update](https://cloud.githubusercontent.com/assets/557895/14926942/c665addc-0e03-11e6-8c53-630ddf055798.png)

![browser_console_and_new_issue_ _mozilla_embedly-proxy_and_firefox_test_pilot_-_activity_stream](https://cloud.githubusercontent.com/assets/557895/14927072/8e3bceea-0e04-11e6-8ae4-6ff656977c61.png)

### Expected results:
No HTTP 500 errors in console.

**Comment by jaredkerim** [comment]
I believe this is what is causing https://github.com/mozilla/activity-stream/issues/518

**Comment by sarracini** [comment]
fixed with https://github.com/mozilla/activity-stream/pull/697

**Comment by jaredkerim** [comment]
This is closed on the server side by #74
